plone/app/portlets/portlets/navigation.py

```diff
@@ -217,11 +217,11 @@ def root_item_class(self):
             return 'navTreeCurrentItem'
         return ''
 
-    def root_icon(self):
-        plone_layout = getMultiAdapter((self.context, self.request),
-            name='plone_layout')
-        icon = plone_layout.getIcon(self.getNavRoot())
-        return icon
+#     def root_icon(self):
+#         plone_layout = getMultiAdapter((self.context, self.request),
+#             name='plone_layout')
+#         icon = plone_layout.getIcon(self.getNavRoot())
+#         return icon
```

Comment:
Instead of commenting or removing this, we prefer to use ``zope.deprecation`` and deprecate it for a while.
An example can be found here
https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/utils.py#L178